src/System.Net.Requests/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true, MaxParallelThreads = 1)]
```

Comment:
> We cannot do that when these tests are run against NETFX.

We could compile this xunit directive into only the netfx test build then, if it makes a measurable impact on test run time.

> if the default value for ServicePointManager.DefaultConnectionLimit is left as 2

We could also increase this value in the various test class' static cctors.

> The problem is that when the tests are run in parallel, it leaves too many connections opern at once which causes the subsequent tests to hang due to not being able to get a new connection.

Why do they hang rather than just queue up to run when the older connections complete?  Or do the tests end up leaving a whole bunch of connections permanently blocked?